libs/blocks/global-navigation/global-navigation.js

```diff
@@ -161,6 +172,8 @@ class Gnav {
       await yieldToMain();
       await task();
     }
+
+    document.addEventListener('click', (e) => closeOnClickOutside(e));
```

Comment:
`document.addEventListener('click', closeOnClickOutside);` should do the same thing